src/core/components/parameter-row.jsx

```diff
@@ -322,7 +321,7 @@ export default class ParameterRow extends Component {
             { !required ? null : <span>&nbsp;*</span> }
           </div>
           <div className="parameter__type">
-            { typeLabel }
+            { schemaObjectType }
```

Comment:
We should be using `schemaObjectTypeLabel` here
```suggestion
   const schemaObjectTypeLabel = fn.getSchemaObjectTypeLabel(schema)
   // ...
   { schemaObjectTypeLabel }
```